index.html

```diff
@@ -0,0 +1,163 @@
+            <input type="number" name="age" id="number" min="0" max="100" class="form-control" placeholder="Age" />
+        </div>
+
+
+        <div class="form-group">
+            <p>Which best describes your agency?</p>
+            <select id="dropdown" name="role" class="form-control" required>
+        <option disabled selected value>Select current role</option>
+        <option value="student">Website Development</option>
+        <option value="fulljob">Marketing</option>
+        <option value="fulljob">Digital Marketing</option>
+        <option value="other">Other</option>
+        </select>
+        </div>
+
+        <div class="form-group">
+            <p>Would you recommend Website Builders America to a friend?</p>
+            <label>
+          <input
+                 name="user-recommend"
+                 value="definitely"
+                 type="radio"
+                 class="input-radio"/>Definitely</label>
+
+            <label>
+          <input
+                 name="user-recommend"
+                 value="maybe"
+                 type="radio"
+                 class="input-radio"/>Maybe</label>
+
+            <label>
+          <input
+                 name="user-recommend"
+                 value="not-sure"
+                 type="radio"
+                 class="input-radio"/>Not sure</label>
+        </div>
+
+        <div class="form-group">
+            <p>What is your favorite feature of Website Builders America?</p>
+            <select id="most-like" name="mostlike" class="form-control" required>
+            <option disabled selected value>Select an option</option>
+            <option value="challenges">Challenges</option>
+            <option value="projects">Projects</option>
+            <option value="community">Community</option>
+            <option value="opensource">Open Source</option>
+            <option value="community">Other</option>
+            </select>
+        </div>
+
+
+        <div class="form-group">
+            <p>What would you like to see improved?<span class="clue">(Check all that apply)</span>
+            </p>
+
+            <label>
+          <input
+                 name="improve"
+                 value="consulting"
+                 type="checkbox"
+                 class="input-checkbox"/>Consulting</label>
+
+            <label>
+          <input
+                 name="improve"
+                 value="design"
+                 type="checkbox"
+                 class="input-checkbox"/>Design
+          </label>
+
+            <label>
+          <input
+                 name="improve"
+                 value="logo"
+                 type="checkbox"
+                 class="input-checkbox"/>logo
+          </label>
+
+            <label>
+          <input
+                 name="improve"
+                 value="price"
+                 type="checkbox"
+                 class="input-checkbox"/>Price
+          </label>
```

Comment:
1. You have too many extra spaces between your code. Try and use linter fixers to solve this.